ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -226,8 +226,10 @@ oc kustomize ${BASE_DIR}/install_yamls/out/openstack/openstack/cr/ | oc apply -f
 sleep 60
 
 # Waiting for all services to be ready
-oc get OpenStackControlPlane openstack -o json | jq -r '.status.conditions[].type' | \
-timeout ${TIMEOUT_SERVICES_READY} xargs -d '\n' -I {} sh -c 'echo testing condition={}; oc wait openstackcontrolplane.core.openstack.org/openstack --for=condition={} --timeout=-1s'
+OPENSTACK_CONTROL_PLANE=$(oc get OpenStackControlPlane -o name)
```

Comment:
depending on the sample config file used, we end up with `openstack` or `openstack-galera` or `openstack-network-isolation`